analyzer/src/funTest/assets/projects/synthetic/spdx/package/libs/curl/package.spdx.yml

```diff
@@ -18,7 +18,7 @@ packages:
      IMAP, SMTP, POP3, RTSP and RTMP. libcurl offers a myriad of powerful features."
   copyrightText: "Copyright (c) 1996 - 2020, Daniel Stenberg, <daniel@haxx.se>, and many
     contributors, see the THANKS file."
-  downloadLocation: "git+github.com:curl/curl.git@53cdc2c963e33bc0cc1a51ad2df79396202e07f8"
+  downloadLocation: "git+github.com/curl/curl.git@53cdc2c963e33bc0cc1a51ad2df79396202e07f8"
```

Comment:
Good finding that this is broken, but IMO it's still not correct. The syntax is described [here](https://spdx.github.io/spdx-spec/3-package-information/#37-package-download-location) to be

    <vcs_tool>+<transport>://<host_name>[/<path_to_repository>][@<revision_tag_or_branch>][#<sub_path>]

So this string should have "https://" added like

    git+https://github.com/curl/curl.git@53cdc2c963e33bc0cc1a51ad2df79396202e07f8

Reply:
And I think you need to reword the commit message, that sentence sounds weird 😄 (and remember s/spdx/SPDX/ 😉 )